### ISSUE TITLE
Line wrapping causes math mode content to overrun the cell

### ISSUE THREAD

**Issue by platformer** [PgBiel/typst-tablex#48]
This works:

```typst
#tablex(
  columns: 3,
  [a: #lorem(7)], [b: $T h i s I s A L o n g A n d R a n d o m M a t h E p r e s s i o n$], [c]
)
```

![image](https://github.com/PgBiel/typst-tablex/assets/40146328/83ea8dbc-602f-4195-b2fd-7466f714c46a)

But making cell `a` line wrap causes strange behavior:

```typst
#tablex(
  columns: 3,
  [a: #lorem(7)], [b: $T h i s I s A L o n g A n d R a n d o m M a t h E p r e s s i o n$], [c]
)
```

![image](https://github.com/PgBiel/typst-tablex/assets/40146328/feaec6e5-1057-4d9b-8c19-b2afe73ef35d)

This is how the native `table` handles it:

```typst
#table(
  columns: 3,
  [a: #lorem(7)], [b: $T h i s I s A L o n g A n d R a n d o m M a t h E p r e s s i o n$], [c]
)
```

![image](https://github.com/PgBiel/typst-tablex/assets/40146328/2cbfa2fa-9cc0-473a-9626-ae6275876608)

One could say it's still suboptimal, but the width of the content is respected.

**Comment by PgBiel** [comment]
Thanks for reporting; I'll take a look when possible.

**Comment by PgBiel** [comment]
Hello, sorry for the delay. The issue is now fixed for the next release (0.0.8). Thanks for reporting!

In the meantime, you can use `tablex.typ` from the [`0.0.x` branch](https://github.com/PgBiel/typst-tablex/tree/0.0.x) - just download the file, place it in your project and import it.
